sideboard/configspec.ini

```diff
@@ -52,18 +60,25 @@ checker.check_skipped_app_config = boolean(default=False)
 [rpc_services]
 ___many___ = string
 
-#[[___many___]]
-#jsonrpc_only = boolean(default=False)
+[[__many__]]
+jsonrpc_only = boolean(default=False)
 
 
 [loggers]
-root = option("DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="DEBUG")
-cherrypy.error = option("DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="DEBUG")
-cherrypy.access = option("DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="DEBUG")
-__many__ = option("DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="DEBUG")
+root = option("TRACE", "DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="DEBUG")
+cherrypy.error = option("TRACE", "DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="DEBUG")
+cherrypy.access = option("TRACE", "DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="CRITICAL")
+__many__ = option("TRACE", "DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="INFO")
```

Comment:
Sideboard actually defines a new logging level above ``DEBUG`` called ``TRACE`` (a common logging level in other languages).  So you can literally say

```
from sideboard.lib import log
log.trace('message which will not appear even at DEBUG level')
```

We don't do this very much, but it's useful for stuff like "show every byte sent and received on a socket" which is too much even for your usual "turn logging up to debug" debugging.